src/components/30-organisms/footer/react/story.jsx

```diff
@@ -29,69 +29,67 @@ storiesOf('Organisms/Footer/React', module)
           onItemClick={link => setNewlyClickedLink(link)}
           clickevents
         >
-          <h2 slot="column-0-title-desktop">axa & you</h2>
-          <h2 slot="column-0-title">axa & you</h2>
+          <h2 slot="column-x-title">axa & you</h2>
           <a
-            slot="column-0-item-0"
+            slot="column-x-item"
             href="https://axa.ch/en/private-customers.html"
             target="_blank"
           >
             Contact
           </a>
           <a
-            slot="column-0-item-1"
+            slot="column-x-item"
             href="https://axa.ch/en/private-customers.html"
             target="_blank"
           >
             Report a claim
           </a>
           <a
-            slot="column-0-item-2"
+            slot="column-x-item"
             href="https://axa.ch/en/private-customers.html"
             target="_blank"
           >
             Broker
           </a>
           <a
-            slot="column-0-item-3"
+            slot="column-x-item"
             href="https://axa.ch/en/private-customers.html"
             target="_blank"
           >
             Job vacancies
           </a>
           <a
-            slot="column-0-item-4"
+            slot="column-x-item"
             href="https://axa.ch/en/private-customers.html"
             target="_blank"
           >
             MyAXA
           </a>
           <a
-            slot="column-0-item-5"
+            slot="column-x-item"
             href="https://axa.ch/en/private-customers.html"
             target="_blank"
           >
             Customer reviews
           </a>
           <a
-            slot="column-0-item-6"
+            slot="column-x-item"
             href="https://axa.ch/en/private-customers.html"
             target="_blank"
           >
             Garage Portal
           </a>
-          <h2 slot="column-1-title-desktop">axa worldwide</h2>
-          <h2 slot="column-1-title">axa worldwide</h2>
+          <h2 slot="column-x-title">axa worldwide</h2>
           <a
-            slot="column-1-item-0"
+            slot="column-x-item"
```

Comment:
I would like to get rid of that X. For social this is possible too, so I would stick to the same idea.